packages/network-controller/tests/provider-api-tests/block-hash-in-response.ts

```diff
@@ -1,7 +1,5 @@
-/* eslint-disable no-loop-func */
```

Comment:
The extension has some of these `eslint-disable` lines but it really shouldn't. I added a `.eslintrc.js` local to `network-controller` to get rid of them.

Besides this, this file should now match the same one in the extension.